### ISSUE TITLE
Reflect the existence of tags in the time entries log 

### ISSUE THREAD

**Issue by heytherewill** [toggl/android#83]
## Description
We need to ensure that time entries (both groups and flat ones) display the little tag icon when they have any number of tags associated to them.

## Dependencies
Depends on #85

## Definition of done
- [x] When a time entry has any tag,  the view that indicates it in the log is visible
- [x] When a time entry has no tags, the view that indicates it is hidden
- [x] *STRETCH GOAL* For performance reasons, it might make sense to implement the billable view as a ViewStub, to prevent inflating it all the time. Let's not do it if it's not worth the hassle, though
